config/variables/stacks/back-office/dev.hcl

```diff
@@ -1,4 +1,10 @@
 locals {
+  azuread_auth_case_officer_group_id      = "c921094a-318f-4996-be5e-9bd2ef9b7bdf"
+  azuread_auth_client_id                  = "2dc22997-3900-4602-b3bd-46385b60e2b2"
+  azuread_auth_enabled                    = true

```

Comment:
As per Radu's comments I don't think this needs to be set as a variable for the component/stack. Can simply just pass in `true` to the module :)

Reply:
Fixed in https://github.com/Planning-Inspectorate/infrastructure-environments/pull/80/commits/cb7ea0166ba6669b373478723805838979484c04